packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -193,16 +194,38 @@ class ContainerPlugin {
         {
           name,
           filename,
-          runtime,
+          runtime: hasSingleRuntimeChunk ? false : runtime,
           library,
         },
         (error: WebpackError | null | undefined) => {
-          if (error) {
-            return callback(error);
+          if (error) return callback(error);
+          if (hasSingleRuntimeChunk) {
+            // Add to single runtime chunk as well.
+            // Allows for singleton runtime graph with all needed runtime modules for federation
+            addEntryToSingleRuntimeChunk();
+          } else {
+            callback();
           }
-          callback();
         },
       );
+
+      // Function to add entry for undefined runtime
+      const addEntryToSingleRuntimeChunk = () => {
+        compilation.addEntry(
+          compilation.options.context || '',
+          //@ts-ignore
+          dep,
+          {
+            name: name ? name + '_partial' : undefined, // give unique name name
+            runtime: undefined,
```

Comment:
produce another entry with _partial for its name, and allow it to be a entrypoint that pushes containerEntryModule into an existing webpack-runtime.js chunk.